README.md

```diff
@@ -53,7 +53,7 @@ Before committing, run the following commands:
 
 or, to combine all three, simply run `./manage.py precommit`.
 
-You can also set up `pylint`, `isort`, `black` and `prettier` in your IDE to avoid doing this manually all the time.
+You can also set up `ruff`, `isort`, `black` and `prettier` in your IDE to avoid doing this manually all the time.
```

Comment:
The changes here still assume a replacement of pylint, but I think you went with running both now.

Reply:
yes, but I'm still undecided. What do you think?

Reply:
Proposal for migration:
* Add necessary configuration for ruff
* Additionally run ruff in CI, but keep pylint as the "official" linter (-> revert changes to the README here)
* monitor what they find in CI and check if we'd miss stuff if we disabled pylint
* -> If we don't, we can disable pylint some time later.